<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>490</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>425</width>
     <height>651</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#widget {
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(102, 0, 255, 	255), stop:1 rgba(181, 0, 255, 255));
}</string>
   </property>
   <widget class="QLabel" name="titel1_2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>30</y>
      <width>251</width>
      <height>45</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 28pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Roblox Account</string>
    </property>
   </widget>
   <widget class="QPushButton" name="checkerButton">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>290</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 0, 115);
font: 16pt &quot;MS Shell Dlg 2&quot;;
border-radius: 20px;</string>
    </property>
    <property name="text">
     <string>Account Checker</string>
    </property>
   </widget>
   <widget class="QPushButton" name="genButton">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>220</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 0, 115);
font: 16pt &quot;MS Shell Dlg 2&quot;;
border-radius: 20px;</string>
    </property>
    <property name="text">
     <string>Account Gen</string>
    </property>
   </widget>
   <widget class="QPushButton" name="creditsButton">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>360</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 0, 115);
font: 16pt &quot;MS Shell Dlg 2&quot;;
border-radius: 20px;</string>
    </property>
    <property name="text">
     <string>Credits</string>
    </property>
   </widget>
   <widget class="QLabel" name="copyright">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>470</y>
      <width>135</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Â© 2021 Warshipping & Liggmar</string>
    </property>
   </widget>
   <widget class="QLabel" name="titel1">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>80</y>
      <width>161</width>
      <height>45</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 28pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Generator</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
